compiler/runtime/OMRCodeCacheManager.hpp

```diff
@@ -171,6 +171,24 @@ class OMR_EXTENSIBLE CodeCacheManager
                                 bool needsToBeContiguous,
                                 bool isMethodHeaderNeeded=true);
 
+   /**
+    * @brief Allocate and initialize a new code cache from a new memory segment
+    *
+    * @param[in] segmentSizeInBytes : segment size to create the code cache from
+    * @param[in] reservingCompilationTID : thread id of the requestor
+    *               if (reservingCompilationTID)
+    *                  >= -1 : the new code cache will be reserved
+    *                  <= -2 : no reservation is requested
+    *                  == -1 : the application thread is requesting the allocation
+    *                  >=  0 : a compilation thread is requesting the allocation
```

Comment:
I find listing `>= -1 : the new code cache will be reserved` a bit confusing. Could use something like this instead:

```
    *                  <= -2 : no reservation is requested
    *                  == -1 : the application thread is requesting the allocation; new code cache will be reserved
    *                  >=  0 : a compilation thread is requesting the allocation; new code cache will be reserved
```

Reply:
Agreed.  Fixed and pushed.